src/modules/roc_fec/reader.cpp

```diff
@@ -226,14 +225,6 @@ bool Reader::check_packet_(const packet::PacketPtr& pp, size_t pos) {
         return (alive_ = false);
     }
 
-    if (pp->rtp()->seqnum != packet::seqnum_t(cur_block_sn_ + pos)) {
```

Comment:
We can compensate removal of this check by adding a new check to rtp::Validator. There we can check that sequence numbers always increase and drop packets violating this rule. There is a validator in the pipeline after fec reader.

Reply:
Should timestamps also increase?

Reply:
Should timestamps also increase?

Reply:
I also want to expand the log msgs produced by validator to:

```
[debug] roc_rtp: rtp validator: too long seqnum jump: prev=150 next=251 dist=101 limit=100
```

As we see that we have `too long jump` but it will be very convenient to see what the maximum allowed number.

Reply:
Timestamps are not required to be monotonic, IIRC. Depacketizer handles this.

Reply:
As discussed we won't handle this case neither in validator nor in reader. We can resolve this discussion.